### PR TITLE
105 - Create/Delete Super Stream

### PR DESCRIPTION
**Pull request by l4mby**
Requires rabbitmq:3.13-rc-management, so that the server can parse the commands

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,5 +53,8 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5",
     "winston": "^3.8.2"
+  },
+  "dependencies": {
+    "semver": "^7.5.4"
```
Comment (tarzacodes) [inline package.json:58]: Adding a dep is quite a commitment, are we 100% we need this?

Comment (l4mby) [reply package.json:58]: We use it to compare the rabbitmq management versions, maybe we can try to hardcode the comparison?

**Review by tarzacodes — APPROVED** [review]
LGTM for me, @icappello ?

**Review by icappello — APPROVED** [review]
looks good to me

**Pull request merged**